### PR TITLE
Support FUSE SDK with dora

### PR DESCRIPTION
**Pull request by LuQQiu**
Add a property key to enable or disable dora metadata cache `alluxio.dora.client.metadata.cache.enabled=true`.

Added full fuse tests with dora data cache only under `alluxio.client.fuse.dor`a.
Added read only from UFS FUSE tests with dora data cache and metadata cache `alluxio.client.fuse.dora.readonly`.
Currently the fuse related tests are duplicated copies with small modification, will refactor the test cases to avoid code copying in the future

### CODE REVIEW

**Comment by LuQQiu** [comment]
@beinan Due to the time consideration, directly copy all fuse tests from fuse unit test to integration tests and connect with dora. All the fuse unit tests passed

File: core/server/worker/src/main/java/alluxio/worker/grpc/DoraWorkerClientServiceHandler.java [hunk 1]
```diff
@@ -124,6 +124,7 @@ public void getStatus(GetStatusPRequest request,
                   .setLastModificationTimeMs(status.getLastModifiedTime())
                   .setOwner(status.getOwner())
                   .setGroup(status.getGroup())
+                  .setCompleted(true)
```
Comment (LuQQiu) [inline core/server/worker/src/main/java/alluxio/worker/grpc/DoraWorkerClientServiceHandler.java:127]: That's the only dora worker related change

**Review by beinan — APPROVED** [review]
lgtm

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Pull request merged**